src/webgl/p5.Camera.js

```diff
@@ -156,7 +176,7 @@ p5.prototype.camera = function (...args) {
  *
  *   rotateX(-0.3);
  *   rotateY(-0.2);
- *   translate(0, 0, -50);
+ *   translate(0, 0, -100);
```

Comment:
We might need to play around with this some more. It looks like it's getting clipped the other way now:

https://github.com/processing/p5.js/assets/5315059/036cc39c-7927-4b60-9483-e12a7f35cc85